consensus/state.go

```diff
@@ -844,7 +854,9 @@ func (cs *ConsensusState) proposalHeartbeat(height, round int) {
 			ValidatorAddress: addr,
 			ValidatorIndex:   valIndex,
 		}
-		cs.privValidator.SignHeartbeat(cs.state.ChainID, heartbeat)
+		if err := cs.privValidator.SignHeartbeat(cs.state.ChainID, heartbeat); err != nil {
+			panic(err)
```

Comment:
Don't think we want to panic here. Log an error though.

Currently it always returns nil